code/modules/jobs/job_types/civilian.dm

```diff
@@ -154,6 +156,7 @@ Strip out?
 	spawn_positions = 2
 	supervisors = "the executive officer"
 	selection_color = "#dddddd"
+	alt_titles = list("Public Defender","Magistrate","Judge")
```

Comment:
Magistrate and Judge imply some sort of power, I like Public Defender though. Maybe we could work this in but that would require more discussion.